recipes/flex/all/conanfile.py

```diff
@@ -67,24 +64,30 @@ def _configure_autotools(self):
             "ac_cv_func_malloc_0_nonnull=yes", "ac_cv_func_realloc_0_nonnull=yes",
             # https://github.com/easybuilders/easybuild-easyconfigs/pull/5792
             "ac_cv_func_reallocarray=no",
-        ]
-
-        autotools.configure(args=configure_args, configure_dir=self._source_subfolder)
-        return autotools
+        ])
+        at.generate()
 
     def build(self):
-        autotools = self._configure_autotools()
+        apply_conandata_patches(self)
+        autotools = Autotools(self)
+        autotools.configure()
         autotools.make()
 
     def package(self):
-        self.copy("COPYING", src=self._source_subfolder, dst="licenses")
-        autotools = self._configure_autotools()
+        copy(self, "COPYING", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
+        autotools = Autotools(self)
         autotools.install()
-        tools.rmdir(os.path.join(self.package_folder, "share"))
-        tools.remove_files_by_mask(os.path.join(self.package_folder, "lib"), "*.la")
+        rmdir(self, os.path.join(self.package_folder, "share"))
+        rm(self, "*.la", os.path.join(self.package_folder, "lib"))
 
     def package_info(self):
         self.cpp_info.libs = ["fl"]
+        self.cpp_info.set_property("cmake_file_name", "FLEX")
```

Comment:
There should be a find mode module here https://cmake.org/cmake/help/latest/module/FindFLEX.html :)